### ISSUE TITLE
[KPI] Add accessibility Sample

### ISSUE THREAD

**Issue by diogofscmariano** [lumada-design/hv-uikit-react#1233]
**Description**
We want a sample in our documentation which shows how can we have accessibility properties added to describe a KPI. This should be able to happen in the top level, or in either of its elements to enhance its description.

**Acceptance Criteria**
- Add sample for PI which illustrates how to inject accessibility aria attributes
